robohive/physics/mj_sim_scene.py

```diff
@@ -37,6 +37,8 @@ def _load_simulation(self, model_handle: Any) -> Any:
         if isinstance(model_handle, str):
             if model_handle.endswith('.xml'):
                 sim = dm_mujoco.Physics.from_xml_path(model_handle)
+            elif "<mujoco" in model_handle:
```

Comment:
Lets check for string and check for "<mujoco"

Reply:
done!